lib/vtls/vtls.c

```diff
@@ -414,7 +414,14 @@ int Curl_ssl_init(void)
 }
 
 #if defined(CURL_WITH_MULTI_SSL)
+# ifdef _MSC_VER
+#   pragma warning(push)
+#   pragma warning(disable : 4132)
+# endif
 static const struct Curl_ssl Curl_ssl_multi;
+# ifdef _MSC_VER
+#   pragma warning(pop)
+# endif
```

Comment:
Is this an MSVC bug? Can we do a standard C initialization here to avoid silencing warnings?

Reply:
I don't think it msvc bug. But I cannot add standard C initialization because on the case where we don't have multi ssl, and it create a  multi redefinition, in line 1326

Reply:
Have you tried moving the `Curl_ssl_cleanup` function below the initialization (e.g. before the function `multissl_version`), and dropping the declaration from line 417?

Reply:
Yes. and it work. See my  new code.